src/components/Accordion/AccordionSummary/AccordionSummary.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+
+import {KeyCode} from '../../../constants';
+import {Disclosure} from '../../Disclosure';
+import type {DisclosureSummaryRenderFunctionProps} from '../../Disclosure/DisclosureSummary/DisclosureSummary';
+import type {QAProps} from '../../types';
+import {isOfType} from '../../utils/isOfType';
+import {useAccordionAttributes} from '../AccordionContext';
+import {accordionSummaryBlock} from '../constants';
+
+import './AccordionSummary.scss';
+
+type AccordionSummaryProps = QAProps & {
+    children: (
+        props: DisclosureSummaryRenderFunctionProps,
+        defaultSummary: React.ReactElement,
+    ) => React.ReactElement;
+};
+
+export function AccordionSummary(props: AccordionSummaryProps) {
+    const {children, qa} = props;
+    const attributes = useAccordionAttributes();
+
+    const handleKeyDown = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+        if (
+            e.key === KeyCode.ARROW_DOWN ||
+            e.key === KeyCode.ARROW_UP ||
+            e.key === KeyCode.HOME ||
+            e.key === KeyCode.END
+        ) {
+            e.preventDefault();
+            const accordionContainer = e.currentTarget.closest('.g-accordion');
+            if (!accordionContainer) return;
```

Comment:
It's strange that eslint missed this, we always write the entire construction with curly braces